### DISCUSSION TITLE
How to add customize attribute of fetch in hono/client? Like credentials: true

### DISCUSSION THREAD

**Discussion by nhattran998** [honojs/hono#1054] [Q&A] [answered]
We can use Hono Client in FE code, but when I try to add credentials in fetch options, I don't know how. I read your team code base in testing and pass the mock fetch with the response. So anyone knows how to add some property like this in fetch? Thank you

Here the code 

```ts
$post = honoClient.xyz.post

await $post({ query: {}, { fetch: () => fetch() ???? })
```

**Comment by yusukebe** [comment, marked as answer]
Hi @harrytran998 !

You can do it by this way:

```ts
await $get(
  {
    query: {},
  },
  {
    fetch: (req) => {
      return fetch(req, {
        headers: {
          foo: 'bar',
        },
      })
    },
  }
)
```

**Comment by nhattran998** [comment]
Oh - it worked! Thank you so much 🙏
